### PR TITLE
Add support for protocol 748 (1.21.40)

### PR DESCRIPTION
**Pull request by CreeperG16**

### CODE REVIEW

**Comment by pokecosimo** [comment]
dataPaths.json broken, fix it

**Comment by CreeperG16** [comment]
I see... Weird, I used incrementVersion.js so I've no idea why those mistakes are there

**Comment by CreeperG16** [comment]
It seems CI is failing because BDS isn't starting up in time? Not entirely sure what's up with that
```
1.21.40
Starting dump server 1.21.40
Downloading server linux 1.21.40 into bds-
Opening https://minecraft.azureedge.net/bin-linux/bedrock-server-1.21.40.00.zip 1729796437257
Opening https://minecraft.azureedge.net/bin-linux/bedrock-server-1.21.40.01.zip 1729796437995
Opening https://minecraft.azureedge.net/bin-linux/bedrock-server-1.21.40.02.zip 1729796438649
Opening https://minecraft.azureedge.net/bin-linux/bedrock-server-1.21.40.03.zip 1729796439278
Opening https://minecraft.azureedge.net/bin-linux/bedrock-server-1.21.40.04.zip 1729796439911
Opening https://minecraft.azureedge.net/bin-linux/bedrock-server-1.21.40.05.zip 1729796440537
Opening https://minecraft.azureedge.net/bin-linux/bedrock-server-1.21.40.06.zip 1729796441170
Opening https://minecraft.azureedge.net/bin-linux/bedrock-server-1.21.40.07.zip 1729796441796
Error: Server did not start on time (60000ms, now 1729796497313)
    at /home/runner/work/minecraft-data/minecraft-data/bedrock-protocol/tools/startVanillaServer.js:132:11
    at waitFor (/home/runner/work/minecraft-data/minecraft-data/bedrock-protocol/src/datatypes/util.js:30:32)
    at async startServerAndWait (/home/runner/work/minecraft-data/minecraft-data/bedrock-protocol/tools/startVanillaServer.js:[12](https://github.com/PrismarineJS/minecraft-data/actions/runs/11505320172/job/32026763933?pr=930#step:10:13)8:3)
    at async Object.startServerAndWait2 (/home/runner/work/minecraft-data/minecraft-data/bedrock-protocol/tools/startVanillaServer.js:[13](https://github.com/PrismarineJS/minecraft-data/actions/runs/11505320172/job/32026763933?pr=930#step:10:14)9:12)
    at async dump (/home/runner/work/minecraft-data/minecraft-data/bedrock-protocol/tools/genPacketDumps.js:28:18)
    at async startTest (/home/runner/work/minecraft-data/minecraft-data/bedrock-protocol/test/internal.js:[16](https://github.com/PrismarineJS/minecraft-data/actions/runs/11505320172/job/32026763933?pr=930#step:10:17):3)
```

File: data/bedrock/latest/proto.yml [hunk 1]
```diff
@@ -652,7 +649,7 @@ packet_move_player:
          source_entity_type: LegacyEntityType
    tick: varint64
 
-packet_rider_jump:
+packet_passenger_jump:
```
Comment (extremeheat) [inline data/bedrock/latest/proto.yml:652]: Please don't rename packets unless it's a different packet (same for fields holding same data), this unnecessarily breaks code compatibility

Comment (extremeheat) [reply data/bedrock/latest/proto.yml]: But if there does need to be a rename for something, it should be backported to the old versions. I've fixed this one.

**Comment by extremeheat** [comment]
Thanks for working on this!

**Pull request merged**